cmd/cli/trafficpolicy_check.go

```diff
@@ -52,30 +58,42 @@ func newTrafficPolicyCheck(out io.Writer) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "check-pods SOURCE_POD DESTINATION_POD",
+		Use:   "check-pods SOURCE_POD DESTINATION_POD [MESH_CONFIG_NAME]",
```

Comment:
Let this not be a positional argument, instead define a flag `--mesh-config-name`

Reply:
Similar as above. Will remove this.